scene/gui/flow_container.cpp

```diff
@@ -155,6 +155,28 @@ void FlowContainer::_resort() {
 			line_data = lines_data[current_line_idx];
 		}
 
+		// The first child of each line adds the offset caused by the alignment,
+		// but only if the line doesn't contain a child that expands.
+		if (child_idx_in_line == 0 && Math::is_equal_approx(line_data.stretch_ratio_total, 0)) {
+			int align_ofs = 0;
+			switch (align) {
+				case ALIGN_BEGIN:
+					break;
```

Comment:
Just a small nitpick, is there a reason why you didn't go with `default: break;` like in the 4.x PR?

Reply:
This is also how I first approached it in Godot 4, but the compiler warned about not having a default option. Here I saw that it is implemented like this in BoxContainer and I also think it reads better. I could change it back if it is desired though.

Reply:
No need, I think it's fine.